go/exocom/src/cmd/index.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "exocom",
+	Short: "",
+	Run: func(cmd *cobra.Command, args []string) {
+		port := os.Getenv("PORT")
+		if port == "" {
+			port = "3100"
+		}
+		fmt.Printf("ExoCom online at port %s\n", port)
+		err := http.ListenAndServe(fmt.Sprintf(":%s", port), nil)
+		if err != nil {
+			panic(err)
```

Comment:
panic includes the stack trace though. This is an unexpected error so I'd prefer to have the stack trace. Anytime where its an error we are planning for or know might fail, I think log.Fatal makes sense as we don't want to give the user the stack trace